sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallAutomationClient.java

```diff
@@ -66,32 +66,32 @@ public Response<CreateCallResult> createCallWithResponse(CreateCallOptions creat
      * Answer an incoming call
      *
      * @param incomingCallContext The incoming call context.
-     * @param callbackUri The call back uri. Optional
+     * @param callbackUrl The call back uri. Optional
```

Comment:
Please help remove the "Optional" word just like the one in Async client. Also change the uri to url please

Reply:
Removed the `optional` wording.
The parameter was already `String`